Makefile

```diff
@@ -39,44 +39,44 @@ testing_data:
 
 test: in
 	rm -f .coverage
-	$(NOSETESTS) -a '!ultra_slow_test' mne
+	$(PYTESTS) -a 'not ultra_slow_test' mne
 
 test-verbose: in
 	rm -f .coverage
-	$(NOSETESTS) -a '!ultra_slow_test' mne --verbose
+	$(PYTESTS) -a 'not ultra_slow_test' mne --verbose
 
 test-fast: in
 	rm -f .coverage
-	$(NOSETESTS) -a '!slow_test' mne
+	$(PYTESTS) -a 'not slow_test' mne
 
 test-full: in
 	rm -f .coverage
-	$(NOSETESTS) mne
+	$(PYTESTS) mne
 
 test-no-network: in
-	sudo unshare -n -- sh -c 'MNE_SKIP_NETWORK_TESTS=1 nosetests mne'
+	sudo unshare -n -- sh -c 'MNE_SKIP_NETWORK_TESTS=1 py.test mne'
```

Comment:
I don't understand what this is doing. I runned both nosetests and pytests changing this flag and there's no change in the test collection.